Makefile.am

```diff
@@ -1,8 +1,9 @@
 ## Process this file with automake to produce Makefile.in
+ACLOCAL_AMFLAGS = -I m4
```

Comment:
remove, this is only required for ancient `automake` versions (even though current Automake still shows a warning :roll_eyes:)

Reply:
Done